docs/data_format_changes/i3330-sec-index-on-json.md

```diff
@@ -0,0 +1,3 @@
+# Sec. Index for JSON 
+
+Some test bodies were changed. This change is just to make change detector stop complaining.
```

Comment:
can we provide the changes or are they minor and does not affect  the customer @islamaliev

Reply:
files from `data_format_changes` are not documentation. They serve as a note for our internal tools